adoc/admin-storage-azure.adoc

```diff
@@ -0,0 +1,667 @@
+= Azure Storage
+The Azure cloud provider can be enabled with {productname} to allow Kubernetes pods to use Azure disks or Azure files as data volumes.
+
+This chapter provides the volume usage and description.
+
+Please refer to link:{docurl}single-html/caasp-deployment/#bootstrap[Cluster Bootstrap] on how to setup Azure cloud provider enabled cluster.
+
+== Azure Disk
+
+[[create_azure_disk]]
+=== Create Disk
+. Install Azure CLI on local machine.
++
+For installation instructions, refer to link:https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest[Install the Azure CLI].
```

Comment:
We have `zypper install azure-cli` ^^

Reply:
Link includes instructions how to install with zypper.